multi-cluster-orchestrator/samples/hello_world/2-workload/charts/hello-world-application/templates/applicationset.yaml

```diff
@@ -18,31 +18,42 @@ metadata:
   name: hello-world-applicationset
   namespace: argocd
 spec:
-    generators:
-    - plugin:
-        configMapRef:
-          name: argocd-mco-placement
-        input:
+  generators:
```

Comment:
Using `templatePatch` requires `goTemplate: true`. It has a different syntax e.g. `'{{`{{name}}`}}-hw'` -> `'{{`{{.name}}`}}-hw'`.

Reply:
Sorry I missed it in the first review!

Reply:
No worries - I'm going to test hello world E2E with the newly published charts (which are more critical), so I find anything else I can include the in the same PR.

Reply:
SG. Thank you!

Reply:
https://github.com/GoogleCloudPlatform/gke-fleet-management/pull/64